Makefile

```diff
@@ -28,7 +28,7 @@ install:
 	pipenv install
 
 unit:
-	nosetests -v
+	pipenv run nosetests -v
```

Comment:
Any reason to add this? `pipenv` is actually called in the `ci` task

Reply:
Using the `unit` make target directly fails without this, whereas other make targets such as `make fmt`, `make integration`, and `make lint` do not. I don't feel strongly about this, it just seemed weird that it was the only make target that must be run with `pipenv run` to not error. Should I revert?

Reply:
CI seemed to be unaffected, so it's fine to leave!